_wiki/essential-knowledge/working-with-futures.md

```diff
@@ -58,15 +58,15 @@ Message message = channel.sendMessage("First!").join();
 // join() ensures, that an exception is thrown in case something went wrong
 message.addReaction("👍").join();
 ```
-`join()` can basically be used every time if you don't care that much about your application's performance. So if you're only developing a small bot for your own server and your current goal is to just get it working, it is probably easier to always use `join()` to get the result of a method. You'll usually not run into problems if you don't overuse this method.
+`join()` can basically be used every time if you don't care that much about your application's performance. So if you're only developing a small bot for your own server and your current goal is to just get it working, it is probably easier to always use `join()` to get the result of a method. You'll won't usually run into problems if you don't overuse this method.
```

Comment:
Isn't `You'll won't` wrong? I guess you wanted to write `You won't`?

Reply:
Yep, thanks for catching that.